map_gui/Cargo.toml

```diff
@@ -27,11 +27,10 @@ instant = "0.1.7"
 js-sys = { version = "0.3.47", optional = true }
 log = "0.4.14"
 map_model = { path = "../map_model" }
-reqwest = { version = "0.11.0", optional = true, default-features=false, features=["blocking", "rustls-tls"] }
 serde = "1.0.123"
 sim = { path = "../sim" }
 subprocess = { version = "0.2.6", optional = true }
-tokio = { version ="1.1.1", features=["rt"], optional = true }
+tokio = { version ="1.1.1", features=["rt-multi-thread"], optional = true }
```

Comment:
`rt` was in [1.1.1](https://docs.rs/tokio/1.1.1/tokio/index.html#feature-flags) too. Cargo won't let you specify non-existent features. 

Using the `rt` feature doesn't mean that this code won't/can't be used in a multithreaded runtime - it just means that **this** crate didn't _require_ any of the multi-thread specific features.  

An external caller could use this crate in any runtime - multithreaded or not. Does this PR introduce usage of the rt-multi-thread features to the map-gui - i.e. would it fail to compile if you reverted this line?

The advantage of specifying `rt` here, rather than 'rt-multi-thread` is just that if the end user *isn't* ultimately using the multithreaded runtime, they can compile less code. 

Since we're ultimately using the multithreaded runtime, this is a purely pedantic distinction 🤓, and since I don't think there are any external users of these crates, I'm fine with this change in any case.

Reply:
Hmm, with just `rt`, I'm getting `no function or associated item named `new` found for struct `Runtime` in the current scope`. That's how I wound up checking https://docs.rs/tokio/1.1.1/tokio/runtime/struct.Runtime.html#method.new. I think we _do_ require this, because `FutureLoader` creates a new runtime every call. I think that's why when you introduced this for the census gen, you were considering storing the runtime on `App` and probably getting it with the more typical `#[tokio::main]` attribute. I'll keep that in mind as something we might want to do, even though I'm guessing there's no real issue creating a few runtimes over the course of the app.

Reply:
Ahh ...looking hard into the mirror this morning.